pkg/controller/route/route_controller.go

```diff
@@ -176,7 +189,6 @@ func (rc *RouteController) reconcile(nodes []*v1.Node, routes []*cloudprovider.R
 					klog.Infof("Creating route for node %s %s with hint %s, throttled %v", nodeName, route.DestinationCIDR, nameHint, time.Since(startTime))
 					err := rc.routes.CreateRoute(context.TODO(), rc.clusterName, nameHint, route)
 					<-rateLimiter
-
 					rc.updateNetworkingCondition(nodeName, err == nil)
```

Comment:
It is no longer correct to set this condition based on the outcome of trying to set a single route.
This is a functional correctness issue.